inst/shiny/helpers.R

```diff
@@ -441,3 +441,21 @@ addRowFiltersToSCE <- function(inSCE, paramsReactive) {
   return(inSCE)
 }
 
+#helper function to add notification bar while code is executing
+spinnerShape = "orbit"
+spinnerColor = "gainsboro"
+
+.loadOpen <- function(message) {
```

Comment:
I don't see the code for popping the Console Log up or down in these functions. Is that possible to add in here?